api/spec/requests/api/v1/ingestions_spec.rb

```diff
@@ -38,4 +39,64 @@
                        included_relationships: [:creator]
     end
   end
+
+  context "ingestion processing" do
+    let!(:ingestion) { FactoryBot.create(:ingestion) }
+    let!(:project_id) { ingestion.project_id }
+
+    before do
+      stub_request(:get, "http://example.com/index.md").
+        to_return(status: 200, body: "", headers: {})
+    end
+
+    it "can start ingestion processing" do
+      patch "/api/v1/ingestions/#{ingestion.id}/process_ingestion"
+      perform_enqueued_jobs
```

Comment:
As Zach suggested, make separate specs for testing if the jobs are doing what they're supposed to. Whenever testing side effects, especially with requests, use `expect do ...` syntax rather than relying on line by line things. Also, use Rails route helpers rather than constructing paths manually.

This should end up looking like:

```ruby
expect do
  post process_api_v1_ingestion_path(ingestion)
end.to have_enqueued_job(Ingestions::ProcessJob).once

expect(response).to be_successful
```

Reply:
In the job specs, you'd want to do something like:

```ruby
expect do
  described_class.perform_now ingestion, user
end.to change { ingestion.reload.finished? }.from(false).to(true)
```